include/uk/plat/lcpu.h

```diff
@@ -123,7 +123,7 @@ struct ukplat_lcpu_func {
 /**
  * Returns the ID of the current logical CPU
  */
-__lcpuid ukplat_lcpu_id(void);
+__u32 ukplat_lcpu_id(void);
```

Comment:
Indeed, this function needs to use `__u64`, since it returns a physical id.
However, I think it is ok to keep `__lcpuid` as `__u32`, since `__lcpuid` represents an index in the `cpus` array (not an actual physical id), so 32 bits should be enough.

Reply:
Also, we need consistency here since this is a common file, so we should use the same header. Or maybe place it in an `#ifdef` guard to check for the architecture?